src/Listener/SaveSuspensionToDatabase.php

```diff
@@ -20,6 +21,21 @@ class SaveSuspensionToDatabase
     use AssertPermissionTrait;
 
     /**
+     * @var SuspendValidator
+     */
+    protected $validator;
+
+    /**
+     * SaveSuspensionToDatabase constructor.
```

Comment:
It could have received a meaningful description, I agree. However these "pointless" phpdocs do have a very important use. Smart IDE's will assist you with auto completion if you have them, if not there's no way of knowing what class it is unless you fully backtrace into the code. Laravel does this too, you know, they phpdoc everything, even if it's only a class reference.